test/card_date/card_date_test.exs

```diff
@@ -73,4 +79,14 @@ defmodule ElixirTools.CardDateTest do
       end
     end)
   end
+
+  test "Various invalid inputs for: from_date!" do
+    invalid_card_dates = [12, "12.22", "12-22", "2022-12-01", "2022/12/01", "02 22", %{}, [], nil]
```

Comment:
can you add a date with month `13` to the mix?

Reply:
how? Literally the integer `13?`